ci-operator/step-registry/openshift-extended/test/longduration/openshift-extended-test-longduration-commands.sh

```
@@ -265,22 +265,88 @@ if [[ $IS_ACTIVE_CLUSTER_OPENSHIFT != "false" ]]; then
 fi
 
-#if OVERWRITE_OC_MIRROR then overwrite the oc-mirror from the payload
+#if OVERRIDE_OC_MIRROR then download oc-mirror from mirror.openshift.com
 if [[ $OVERRIDE_OC_MIRROR == "true" ]]; then
     echo "OCP Version: ${ocpVersion}"
-    if [[ "$ocpVersion" == *arm* ]] || [[ "${OCP_ARCH:-}" != "amd64" ]]; then
-        echo "[WARN]OCP_ARCH is not amd64, or OCP is not for amd64, currently do not support to overwrite the oc-mirror from the OCP release image"
-    fi
     if [[ -n "${ocpVersion:-}" ]]; then
         tmpDir=$(mktemp -d)
         cd ${tmpDir}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)